client/fgfs-addon/addon-main.nas

```diff
@@ -7,6 +7,56 @@
 # @author Colin Geniet, 2021
 
 
+var GenericRadio = {
+    # Parameter: root: the radio property root (e.g. /instrumentation/comm[i]), as a property node.
+    new: func(root) {
+        var r = { parents: [GenericRadio], root: root, };
+        r.init();
+        return r;
+    },
+
+    init: func {
+        # Radio frequencies are initialized by C++ code even for aircrafts which do not use the radio.
+        # To avoid transmitting the frequency for an unused radio (which would make it functional
+        # in the fgcom-mumble plugin), test if the 'operable' property exist.
+        # It is created by the C++ instrument code, if the radio is used.
+        me.operable = me.root.getNode("operable");
+        me.is_used = (me.operable != nil);
```

Comment:
Consider publishing this as property in the fgcom-mumble subnode, so the combar can pick it up easily :)